include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -296,6 +300,30 @@
 #define _ONEDPL_CPP20_SHIFT_LEFT_RIGHT_PRESENT                                                                         \
     (_ONEDPL___cplusplus >= 202002L && (_MSC_VER >= 1921 || _GLIBCXX_RELEASE >= 10))
 
+#if _ONEDPL___cplusplus >= 202002L && __cpp_concepts >= 201907L && __cpp_lib_concepts >= 202002L
+#    define _ONEDPL_CPP20_CONCEPTS_PRESENT 1
+#else
+#    define _ONEDPL_CPP20_CONCEPTS_PRESENT 0
+#endif
+
+#if _ONEDPL___cplusplus >= 202302L && __cpp_lib_tuple_like >= 202207L
+#    define _ONEDPL_CPP23_TUPLE_LIKE_COMMON_REFERENCE_PRESENT 1
+#else
+#    define _ONEDPL_CPP23_TUPLE_LIKE_COMMON_REFERENCE_PRESENT 0
+#endif
+
+#if _ONEDPL___cplusplus >= 202302L && __cpp_lib_ranges_zip >= 202110L
+#    define _ONEDPL_CPP23_RANGES_ZIP_PRESENT 1
+#else
+#    define _ONEDPL_CPP23_RANGES_ZIP_PRESENT 0
+#endif
```

Comment:
```suggestion
#if _ONEDPL_STD_FEATURE_MACROS_PRESENT
#    define _ONEDPL_CPP20_CONCEPTS_PRESENT (__cpp_concepts >= 201907L && __cpp_lib_concepts >= 202002L)
#    define _ONEDPL_CPP23_TUPLE_LIKE_COMMON_REFERENCE_PRESENT (_ONEDPL___cplusplus >= 202302L && __cpp_lib_tuple_like >= 202207L)
#    define _ONEDPL_CPP23_RANGES_ZIP_PRESENT (_ONEDPL___cplusplus >= 202302L && __cpp_lib_ranges_zip >= 202110L)
#else
#    define _ONEDPL_CPP20_CONCEPTS_PRESENT 0
#    define _ONEDPL_CPP23_TUPLE_LIKE_COMMON_REFERENCE_PRESENT 0
#    define _ONEDPL_CPP23_RANGES_ZIP_PRESENT 0
#endif
```

Reply:
Thanks, I have switched to defining them this way.